tests/unit/test_connection.py

```diff
@@ -41,7 +41,7 @@ def test_connection_retry(
         )
 
         # Act
-        with pytest.raises(FailedToConnectException):
+        with pytest.raises(FailedToConnectError):
```

Comment:
Should this be the dbtRuntimeError? Based on [this documentation](https://docs.getdbt.com/guides/best-practices/debugging-errors), failing to connect to the Dremio instance should result in a dbtRuntimeError right? Doe we have to change what error is raised when connecting?

Reply:
I'll change to dbtRuntimeError and test. Thanks for the suggestion.